src/components/filter/FilterWrapper.tsx

```diff
@@ -44,11 +43,9 @@ export const FilterWrapper = ({
   handleToggle,
   hasFilter,
   label,
-  ref,
   style,
```

Comment:
You can use React.forwardRef to use ref unless we don't have anywhere pass the ref down to this component and you want to take the ref out completely.

Reply:
Yeh it wasn't being used at all apparently